dimod/bqm/shapeablebqm.pyx.src

```diff
@@ -914,6 +914,52 @@ cdef class cyAdj@name@BQM:
 
         self.bqm_.set_linear(vi, b)
 
+    def add_constraint_as_quadratic(self, object terms, Bias
+                                    lagrange_multiplier, Bias constant):
```

Comment:
Doesn't have to be implemented here, but let's plan to have (in version 1) an adaptive default value for `lagrange_multiplier`. (To avoid chain strength-like issues)

Reply:
I think for now, I prefer the explicit value. I know it passes complexity to the user, but for now when we're lacking best practices to me it makes sense.

Reply:
But totally agree we would want that in the long run.

Reply:
The tuning of `lagrange_multiplier` is not handled by the BQM class. It must be an explicit value.